plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -45,6 +45,11 @@ library
     , process-extras  >= 0.7.1
     , text
 
+  if impl(ghc >= 9.2)
+    build-depends: fourmolu ^>= 0.8.2
+  if impl(ghc >= 9.4)
+    build-depends: fourmolu ^>= 0.9
+  
```

Comment:
I'm a bit confused by exactly why this is done this way (same as https://github.com/haskell/haskell-language-server/pull/3364#discussion_r1031792168, basically). But if it passes CI I'm fine with it.

I'll revisit all these bounds next time I come along to bump the version.

Reply:
> I'm a bit confused by exactly why this is done this way

Mainly, because I don't know any better. It seemed to be a crude but correct way to get tests pass.

I'll be happy if somebody with more knowledge fixes this the right way.